Integrations/rasterize/rasterize.py

```diff
@@ -66,22 +67,23 @@ def init_driver(offline_mode=False):
     return driver
 
 
-def rasterize(path: str, width: int, height: int, r_type='png', offline_mode=False):
+def rasterize(path: str, width: int, height: int, r_type: str = 'png', wait_time: int = 0, offline_mode: bool = False):
     """
     Capturing a snapshot of a path (url/file), using Chrome Driver
     :param offline_mode: when set to True, will block any outgoing communication
     :param path: file path, or website url
     :param width: desired snapshot width in pixels
     :param height: desired snapshot height in pixels
     :param r_type: result type: .png/.pdf
+    :param wait_time: time in seconds to wait before taking a screenshot
     """
     driver = init_driver(offline_mode)
 
     try:
         demisto.debug(f'Navigating to path. Mode: {"OFFLINE" if offline_mode else "ONLINE"}')
 
         driver.get(path)
-        driver.implicitly_wait(5)
+        time.sleep(wait_time) if wait_time > 0 else driver.implicitly_wait(5)  # type: ignore
```

Comment:
I think the sleep should be done after the `implicitly_wait`. So we first make sure the page loaded and then if needed add another sleep to allow the page to fully render (Javascript stuff).